lua/nvim-tree/utils.lua

```diff
@@ -388,6 +388,23 @@ function M.array_remove_nils(array)
   end, array)
 end
 
+--- create a new deep cloned table
+--- @param t table
+--- @return table
+function M.table_deep_clone(t)
```

Comment:
Could we use `vim.deepcopy()` instead of adding that until function?

Reply:
Thank you! I searched for deep_copy...